app/lib/utils/translations/translations_transaction.dart

```diff
@@ -98,11 +172,42 @@ class TranslationsFrTransaction implements TranslationsTransaction {
       "Tu t'es désinscrit du prochain cycle. Tu peux te réinscrire si tu changes d'avis.";
   @override
   String get unregisterParticipantNotificationTitle => 'Inscription annulée';
+  @override
+  String get alreadyEndorsedErrorBody => 'Ce compte a déjà été endossé pour ce cycle';
+  @override
+  String get alreadyEndorsedErrorTitle => 'Déjà endossé';
+  @override
+  String get noValidClaimsErrorBody =>
+      "Vous n'avez pas envoyé de confirmations valables. As-tu scanné les autres participants?";
+  @override
+  String get noValidClaimsErrorTitle => 'Pas de confirmations valides';
+  @override
+  String get votesNotDependableErrorBody =>
+      "Seule la moitié ou moins des participants attribués ont été confirmés, ou certains participants n'ont pas encore envoyé leurs confirmations. Cela empêche le paiement immédiat du revenu et tu dois attendre 48 heures.";
+  @override
+  String get votesNotDependableErrorTitle => 'Confirmations insuffisantes';
+  @override
+  String get insufficientFundsErrorBody =>
+      "Tu n'as pas assez d'argent sur ce compte. Regarde sur le site web de ta communauté locale pour savoir comment en obtenir.";
+  @override
+  String get insufficientFundsErrorTitle => 'Trop peu de crédit';
+  @override
+  String get txTooLowPriorityErrorBody =>
+      "Erreur technique de transaction. Cela peut arriver si tu envoies deux fois une transaction dans un laps de temps très court. Veuillez patienter quelques secondes'.";
+  @override
+  String get txTooLowPriorityErrorTitle => 'Erreur de priorité de transaction';
+  @override
+  String get rewardsAlreadyIssuedErrorBody =>
+      'Un autre participant a déclenché le paiement pour cette rencontre. Tu devrais avoir avoir déjà reçu ton revenu.';
```

Comment:
Another typo in my text. :)
```suggestion
      'Un autre participant a déclenché le paiement pour cette rencontre. Tu devrais avoir déjà reçu ton revenu.';
```